components/brave_wallet_ui/utils/tx-utils.ts

```diff
@@ -422,6 +422,11 @@ export const findTransactionToken = <
   )
 }
 
+// Use this function to synchronously extract swap/bridge details from a
+// transaction.
+//
+// Prefer using useSwapTransactionParser() hook in React components, which
+// can asynchronously extract details from the blockchain.
```

Comment:
Mind using `/** */` style comments here so the notes appear when the function is hovered-over in a text editor?